src/components/Navigation.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  changeFunction: (event: React.MouseEvent) => void
+};
+
+export const Navigation: React.FC<Props> = ({ changeFunction }) => {
+  const [active, setActive] = useState('All');
```

Comment:
It's looks like this state already exist in `App` component, you can pass as props `status` from `App` and use it in this component